alerts/src/commonMain/kotlin/Alerts.kt

```diff
@@ -26,30 +26,33 @@ import kotlinx.coroutines.suspendCancellableCoroutine
 import kotlin.coroutines.resume
 
 typealias AlertActionHandler = () -> Unit
+typealias AlertTextWatcher = (String) -> Unit
```

Comment:
I thing string should be optional (if nothing entered it doesn't mean empty string)

Reply:
Not blocking but usually we use `Observer` instead of `Watcher`

Reply:
Android has always default text empty and testing on iOS it seems like when I remove all text it gives back empty and not null.

Reply:
It's also easier to handle as empty text as rarely you want to also handle null.